phys/module_surface_driver.F

```diff
@@ -3861,6 +3867,186 @@ SUBROUTINE surface_driver(                                         &
 
 ! end of CLM scehme
 ! -------------------------------------------------------------------
+#endif
+
+#ifdef WRF_USE_CTSM
+     CASE (CTSMSCHEME)
+
+! TODO (NS): Adding errors and warning on compatible PBL schemes, etc. For
```

Comment:
@negin513 
Negin,
Has this been moved into check_a_mundo? This "TODO" should be removed.

Reply:
@davegill No. This has not been moved to check_a_mundo. I put this here similar to what was there for other land models for example: 
https://github.com/wrf-model/WRF/blob/94df8047e63c91e147c1c7ce803485e2d7656e51/phys/module_surface_driver.F#L3631-L3635

Reply:
I just removed the TODO comment from this file.

Reply:
Negin,
All good - I just wanted that "TODO" thing removed.